spec/controllers/curation_concerns/file_sets_controller_spec.rb

```diff
@@ -164,8 +170,6 @@
           expect(CharacterizeJob).to receive(:perform_later).with(file_set.id, kind_of(String))
           post :update, id: file_set, file_set: { files: [file] }
           expect(response).to redirect_to main_app.curation_concerns_file_set_path(file_set)
-          skip 'pending hydra-works#89'
-          expect(file_set.reload.label).to eq 'image.png'
```

Comment:
Can you comment on this deletion? Would this expectation fail?

(To be clear, I'm not even sure why we'd need to check a label here.)